agenttest.py

```diff
@@ -332,21 +332,20 @@ def send_arps(hosts):
 # pylint: disable=too-many-locals, too-many-statements
 
 
-def end_to_end_test():
+def end_to_end_test(options, envs):
```

Comment:
The right thing to do here is to pass in the arguments as parameters with defaults, e.g.
`cert_dir=CERT_DIR`